duckduckgo/iOS
# new sync flow

Task/Issue URL: https://app.asana.com/0/1204186595873227/1205343018025256/f

**Description**: Update sync flow according to https://www.figma.com/file/K7uUDaJC76zr33VpilBvp4/%F0%9F%94%84-Sync-7?node-id=6715%3A125192&mode=dev



**Steps to test this PR**:
Test all scenarios in the Flow (like in the figma file) check everything works correctly and that design and copy matches.

Please though note the following regarding the sync confirmation screen:

1. When creating an account through single device setup, never show options.
2. When creating an account via the Connect endpoint, always show these options, but only on one of the devices. Pick the device that is doing the scanning, or, in which the code is being pasted.
3. When signing in via the login endpoint, show these options if and only if the device logging in is the second device of an account.
https://app.asana.com/0/0/1205658509456648/f
4. When 2 devices are connected it should show the name of the other device otherwise should tell the number of devices sync (this is indicated in the design as well)

<!—
Before submitting a PR, please ensure you have tested the combinations you expect the reviewer to test, then delete configurations you *know* do not need explicit testing.

Using a simulator where a physical device is unavailable is acceptable.
—>

**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


—
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
